src/database/migrations/20230131072323-create-user.js

```diff
@@ -1,33 +0,0 @@
-/** @type {import('sequelize-cli').Migration} */
```

Comment:
Why did you delete this file?

Reply:
> Why did you delete this file?

i don't think i did it on purpose but let me check